core/domain/stats_jobs_one_off.py

```diff
@@ -141,12 +141,21 @@ def map(item):
                 else:
                     try:
                         exploration = exp_services.get_exploration_by_id(
-                            item.exploration_id, item.exploration_version)
+                            item.exploration_id,
+                            version=item.exploration_version)
                     except Exception:
                         # Exploration does not exist.
                         return
-                    dest_state = exploration.states[
-                        unicode_state_name].interaction.default_outcome.dest
+
+                    state = exploration.states[unicode_state_name]
+                    if state.interaction.default_outcome is None:
+                        yield (
+                            'LOG: Default outcome does not exist for state %s '
```

Comment:
Do we expect this to happen in practice? If not, change LOG to ERROR

Reply:
Right. changed.